web/packages/teleport/src/Roles/RoleEditor/StandardEditor/AccessRules.tsx

```diff
@@ -112,6 +115,26 @@ const AccessRule = memo(function AccessRule({
         value={verbs}
         onChange={v => setRule({ ...value, verbs: v })}
         rule={precomputed(validation.fields.verbs)}
+      />
+      <FieldInput
+        label="Condition"
+        toolTipContent={
+          <>
+            Additional condition, expressed using the{' '}
+            <Text
+              as="a"
+              href="https://goteleport.com/docs/reference/predicate-language/"
+              target="_blank"
+              color={theme.colors.interactive.solid.accent.default}
+            >
+              Teleport predicate language
+            </Text>
```

Comment:
what about something like: `Optional condition further limiting the scope for this rule expressed using the ....`